File: src/prolog/standard/fizzbuzz.pl
fizzbuzz(N, "FizzBuzz") :- Q is mod(N, 15), Q == 0.
fizzbuzz(N, "Buzz") :- Q is mod(N, 5), Q == 0.
fizzbuzz(N, "Fizz") :- Q is mod(N, 3), Q == 0.
fizzbuzz(N, N).

File: src/main/kotlin/nl/fifthpostulate/standard/FizzBuzz.kt
package nl.fifthpostulate.standard

class FizzBuzz {
    fun of(n: Int): String {
        if (n % 15 == 0) {
            return "FizzBuzz"
        }
        if (n % 5 == 0) {
            return "Buzz"
        }
        if (n % 3 == 0) {
            return "Fizz"
        }
        return n.toString()
    }
}